precommit/src/main/shell/test-patch-docker/Dockerfile

```diff
@@ -324,40 +324,6 @@ RUN apt-get -q update && apt-get -q install --no-install-recommends -y \
         yamllint==1.24.2 \
     && rm -rf /root/.cache \
     && mv /usr/local/bin/pylint /usr/local/bin/pylint3
```

Comment:
I think both
```
    && mv /usr/local/bin/pylint /usr/local/bin/pylint3
```
and
```
RUN ln -s /usr/local/bin/pylint3 /usr/local/bin/pylint
```
can be removed.

Reply:
Taking this out leaves just `/usr/local/bin/pylint` . I kept it like this so that `/usr/local/bin/pylint` is a symlink to `/usr/local/bin/pylint3` (which is the real binary) in case someone wants to easily override the version in the container or add python2 back or for python 4 or whatever.